### ISSUE TITLE
"hammr scan" never starts.

### ISSUE THREAD

**Issue by MaxTakahashi** [usharesoft/hammr#36]
Hi,

"hammr scan" command never starts scan. Stopped at "Searching scan on uforge ..." and never proceed.

```
[root@hammr-351 ~]# hammr scan run --ip 192.168.56.104 --scan-login root --name hmrscan
Password for root@192.168.56.104:
Searching scan on uforge ...
```

Glassfish receives "scannedInstances_get" request repeatedly until the scan was cancelled.

Thanks.

**Comment by segalaj** [comment]
Hello,
Could you please try to reproduce this issue on a 3.6 uforge?
Thanks

**Comment by AliceAlbano** [comment]
Hello,
I think this one is fixed, I tried to reproduce the issue on a 3.7 forge by scanning several machines, and the scan always starts.
